### ISSUE TITLE
Links from `IDA` to Cosmos SDK is broken

### ISSUE THREAD

**Issue by happylolonly** [cosmos/developer-portal#61]
A lot of Cosmos SDK links not working correctly, for example https://docs.cosmos.network/main/core/encoding.html

https://ida.interchain.io/academy/2-cosmos-concepts/6-protobuf.html

<img width="741" alt="image" src="https://github.com/cosmos/developer-portal/assets/18665326/67fa7260-87d3-447b-997d-bc10dbdc3957">

But saw similar in different sections.
As having expertise in front-end, I think maybe it is not only IDA fix, maybe better to handle it by redirecting in Cosmos docs.

**Comment by xavierlepretre** [comment]
Indeed. I have fixed those I could find here https://github.com/cosmos/developer-portal/pull/59
